hummingbot/logger/logger.py

```diff
@@ -51,11 +51,13 @@ def notify(self, msg: str):
             hummingbot_app.notify(f"({pd.Timestamp.fromtimestamp(int(time.time()))}) {msg}")
 
     def network(self, log_msg: str, app_warning_msg: Optional[str] = None, *args, **kwargs):
-        from hummingbot.client.hummingbot_application import HummingbotApplication
+        if app_warning_msg is not None and not HummingbotLogger.is_testing_mode():
```

Comment:
This line below is causing the tests to fail and I had to put it under this condition to not include when in testing mode. The change is not allowed since there is not test coverage around this change so any help would be appreciated. Thx